MoreLinq/Fold.g.tt

```diff
@@ -61,7 +61,14 @@ namespace MoreLinq
 
         public static TResult Fold<T, TResult>(this IEnumerable<T> source, Func<<#= e.Ts #>, TResult> folder)
         {
-            return FoldImpl(source, <#= e.CountArg #>, <#= e.FolderArgs #>);
```

Comment:
Since `FolderArg` is no longer used, it can be removed.

Reply:
Removed.